bin/prod-deploy/aws.sh

```diff
@@ -178,21 +177,6 @@ function getTargetHealth() {
     | jq -r -c '.TargetHealthDescriptions[0].TargetHealth.State'
 }
 
-# Remove an EC2 instance from the ELB target group.
-#
-# $1 - ID of the EC2 instance to remove
-#
-# Expects global environment variables:
-#   AWS_PROD_API_PORT - Port the EC2 instance is listening on
-#   AWS_PROD_API_TARGET_GROUP_ARN - The ARN of the target group to remove the
-#     instance from.
-function removeInstanceFromTargetGroup() {
-  aws elbv2 deregister-targets \
-    --target-group-arn $AWS_PROD_API_TARGET_GROUP_ARN \
-    --targets Id=$1,Port=$AWS_PROD_API_PORT \
-    > /dev/null
-}
-
```

Comment:
Terminating the instance also removes it from the target group. Removing this step should speed up deployments a little bit because it's one less thing we have to wait on.